### ISSUE TITLE
Question about using keras with ranking_head

### ISSUE THREAD

**Issue by CYL586309** [tensorflow/ranking#279]
hi TFR team:
I want to use ranking_head and  keras together for replacing tf.compat.v1.layers in score_fn ([tf_ranking_tfrecord.py](https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/examples/tf_ranking_tfrecord.py)). I notice that tfr.keras.canned.DNNRankingNetwork can build an keras model. Here is my code:
```python
def _score_fn(context_features, example_features, mode, params, config):
    network = tfr.keras.canned.DNNRankingNetwork(
        context_feature_columns=context_features_columns,
        example_feature_columns=example_feature_columns,
        hidden_layer_dims=_HIDDEN_LAYER_DIMS,
        activation=tf.nn.relu,
        dropout=0.8,
        use_batch_norm=True,
        batch_norm_moment=0.99,
        name="dnn_ranking_model")
    is_training = (mode == tf.estimator.ModeKeys.TRAIN)
    logits = network.score(context_features=context_features, example_features=example_features, training=is_training)
    return logits
```
However, the output loss and metrics are all zeros:
```python
INFO:tensorflow:loss = 0.0, step = 0
*** basic_session_run_hooks.py:262] loss = 0.0, step = 0
INFO:tensorflow:Saving dict for global step 500: global_step = 500, labels_mean = 0.0, logits_mean = 0.0, loss = 0.0, metric/arp = 0.0, metric/ndcg@1 = 0.0, metric/ndcg@10 = 0.0, metric/ndcg@3 = 0.0, metric/ndcg@5 = 0.0, metric/ordered_pair_accuracy = 0.0
```
so I'm curious about the way of building DNN model based on keras with ranking_head  in the estimator framework.
Thanks!
